### PR TITLE
migrate_vm: Fix domjobinfo comparison

### PR DESCRIPTION
**Pull request by dzhengfy**
It is unreasonable that the domjobinfo when completion on source host
should be equal to that on target host because some fields may have
differences and other values should be equal on both sides.

Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

**Comment by dzhengfy** [comment]
@will-Do please help review this item

**Comment by dzhengfy** [comment]
@waynesun09 , could you help review this?

File: libvirt/tests/src/migration/migrate_vm.py [hunk 1]
```diff
@@ -865,6 +865,78 @@ def check_iothread_after_migration(vm_name, params, iothread):
...
 
 
+def check_domjobinfo_on_complete(test, source_jobinfo, target_jobinfo):
+    """
+    Check the domjobinfo outputs on source and target hosts.
+    They should be same except following fields:
+        Time elapsed:
+        Time elapsed w/o network
+    Above values on source host should be larger than that on
+    target host with about 2 seconds
```
Comment (will-Do) [inline libvirt/tests/src/migration/migrate_vm.py]: Could you please declare the above desc more clear

File: libvirt/tests/src/migration/migrate_vm.py [hunk 2]
```diff
@@ -865,6 +865,78 @@ def check_iothread_after_migration(vm_name, params, iothread):
...
+    :param test: avocado.core.test.Test object
+    :param local_jobinfo: The domjobinfo output on source host
+    :param remote_jobinfo: The domjobinfo output on target host
+    """
+    source_info = read_domjobinfo(test, source_jobinfo)
+    target_info = read_domjobinfo(test, target_jobinfo)
+
+    for pair in source_info.items():
+        key = pair[0]
+        value = pair[1]
```
Comment (will-Do) [inline libvirt/tests/src/migration/migrate_vm.py]: key, value = pair

File: libvirt/tests/src/migration/migrate_vm.py [hunk 3]
```diff
@@ -865,6 +865,78 @@ def check_iothread_after_migration(vm_name, params, iothread):
...
+    """
+    Read the domjobinfo into a dict
+
+    :param test: avocado.core.test.Test object
+    :param domjobinfo: The domjobinfo command output
+
+    :return: A dict contains the domjobinfo
+    """
+    jobinfo_dict = {}
+    domjobinfo_list = domjobinfo.split("\n")
```
Comment (will-Do) [inline libvirt/tests/src/migration/migrate_vm.py]: splitlines() doese the same thing

File: libvirt/tests/src/migration/migrate_vm.py (hunk 1 above)
Comment (dzhengfy) [inline libvirt/tests/src/migration/migrate_vm.py]: @will-Do updated.

File: libvirt/tests/src/migration/migrate_vm.py (hunk 2 above)
Comment (dzhengfy) [inline libvirt/tests/src/migration/migrate_vm.py]: @will-Do updated.

File: libvirt/tests/src/migration/migrate_vm.py (hunk 3 above)
Comment (dzhengfy) [inline libvirt/tests/src/migration/migrate_vm.py]: @will-Do updated.

**Pull request merged**